### DISCUSSION TITLE
Ubuntu Podman unable to display images from remote push

### DISCUSSION THREAD

**Discussion by PunkFleet** [containers/podman#23117] [Q&A]
I using Podman setup a private registry. and i can push image `registry.xxxxx.com:5000/xxxx/hello` from localhsot to registry.
If i
```
curl -u user:pass https://registry.xxxxx.com:5000/v2/_catalog
{"repositories":["hello","xxxx/hello"]}
```
Great, i see It's success. 
But....
```
podman images
REPOSITORY                           TAG         IMAGE ID      CREATED        SIZE
docker.io/library/registry           latest      6a3edb1d5eb6  8 months ago   26 MB
registry.xxxx.com:5000/hello  latest      d2c94e258dcb  14 months ago  28.5 kB
docker.io/library/hello-world        latest      d2c94e258dcb  14 months ago  28.5 kB
```
There don't display the new image that from localhost pushed.
It should display
```
registry.xxxxx.com:5000/xxxx/hello
```

May i missed something? or this is a bug?

**Comment by Luap99** [comment]
You have to share the full commands that you use, it is not exactly clear what you are doing and what output you expect.
podman images lists images in the local storage which is different from a local registry.

**Reply by PunkFleet** [reply]
i understood what difference `podman` with `skopeo` just now. I want manage private reigistry's images in server local(there also installed podman), i used podman command, but i know is wrong, i should using `skopeo`. but it real hard to manage private registry that setuped by podman.
